File: Homeworks/1. Programming/2. C#-Part-2/03.Methods/02.Get largest number/GetLargestNumber.cs
//Write a program that reads 3 integers from the console and prints the largest of them using the method GetMax().

namespace _02.Get_largest_number
{
    using System;
    using System.Linq;

    class GetLargestNumber
    {
        static int LargestNumber(int[] numbers)
        {
            Array.Sort(numbers);
            return numbers.Last();
        }

        static void Main()
        {
            var input = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();

            Console.WriteLine(LargestNumber(input));
        }
    }
}

File: Homeworks/Web Design and UI technlogies/3. JavaScript-Part-1/06.Functions/02.GetLargestNumber/GetLargestNumber.js
//Write a method GetMax() with two parameters that returns the larger
//of two integers. Write a program that reads 3 integers from the console and prints the largest of them using the method GetMax().

function GetLargestNumber(args) {
    var input = args[0].split(' ').map(Number),
        a = input[0],
        b = input[1],
        c = input[2];

    // max = Math.max(a, b, c);
    if (GetMax(a, b) > c) {
        return GetMax(a, b);
    } else {
        return c;
    }

    function GetMax(first, second) {
        return first > second ? first : second;
    }

    return console.log(max);
}
//test
GetLargestNumber(['8', '3 ', '6']);
GetLargestNumber(['1', '1 ', '3']);